app/controllers/concerns/wizard_steps.rb

```diff
@@ -1,38 +1,13 @@
 module WizardSteps
   extend ActiveSupport::Concern
 
+  include DFEWizard::Controller
```

Comment:
I would have expected this whole concern to be in the gem... did you run into a reason why it can't be?

Reply:
How generic do we think the gem should be? Think it'll be of use to other teams?

Reply:
I was talking to Joe about this earlier, I think our wizard is so coupled to GiT/the API with matchback, magic links, pre-filling etc that trying to pull out a just a basic wizard would mean we don’t get very DRY between the two services (unless we added yet another gem into the mix). I think this wizard should be focused to GiT/API/CRM services, but open to other opinions